blocks.md

```diff
@@ -953,29 +956,25 @@ use_bits = false
 Key | Values | Required | Default
 ----|--------|----------|--------
 `device` | Network interface to monitor (name from /sys/class/net). | No | Automatically chosen from the output of `ip route show default`
-`format` | A string to customise the output of this block. See below for available placeholders. Text may need to be escaped, refer to [Escaping Text](#escaping-text). | No | `"{speed_up} {speed_down}"`
+`format` | A string to customise the output of this block. See below for available placeholders. Text may need to be escaped, refer to [Escaping Text](#escaping-text). | No | `"{speed_up;K} {speed_down;K}"`
 `format_alt` | If set, block will switch its formatting between `format` and `format_alt` on every click. | No | None
-`speed_digits` | Number of digits to use when displaying speeds. | No | `3`
-`speed_min_unit` | Smallest unit to use when displaying speeds. Possible choices: `"B"`, `"K"`, `"M"`, `"G"`, `"T"`. | No | `"K"`
-`use_bits` | Display speeds in bits instead of bytes. | No | `false`
```

Comment:
Need to document that this has been removed. Perhaps better if we show what the new way is as well.

Reply:
Let's document these breaking change in NEWS.md. I started it off with a couple of examples from other blocks.